src/pages/LabEditor/LabEditor.tsx

```diff
@@ -109,21 +109,21 @@ export default function LabEditor({match: {params: {moduleId, labId}}}: Props) {
 
   const [selectedVm, setSelectedVm] = useState<number|undefined>();
   const popover = (
-    <Popover id="popover-basic"
-    style={{
-      maxWidth: '750px'
-    }}>
-      <Popover.Title as="h3">Info:</Popover.Title>
+    <Popover id='popover-basic' style={{maxWidth: '750px'}}>
+      <Popover.Title as='h3'>Info:</Popover.Title>
```

Comment:
I would change the title to be "Lab Types" to be more clear on what info it is displaying.

Reply:
Done. Changed popover title from "Info" to "Lab Types".